constant.py

```diff
@@ -0,0 +1,4 @@
+API_TOKEN = 'token'
```

Comment:
The secret token should not be hardcoded.
You should read it form the config file or fetch it from the [environment variable](https://linuxhint.com/set-environment-variables-python/#:~:text=Read%20Environment%20Variables%20in%20Python,variable%20by%20using%20this%20object.).